src/lib/viewers/image/ImageViewer.js

```diff
@@ -297,6 +316,9 @@ class ImageViewer extends ImageBaseViewer {
             this.annotationControls = new AnnotationControls(this.controls);
             this.annotationControls.init({
                 fileId: this.options.file.id,
+                initialMode: this.options.enableAnnotationsImageDiscoverability
```

Comment:
We also need to init the `AnnotationControlsFSM` with `AnnotationState.REGION` in `ImageViewer` to keep the source of truth up to date